paperqa/docs.py

```diff
@@ -426,17 +503,14 @@ async def aget_evidence(
         if disable_vector_search:
             matches = self.texts
         else:
-            query_vector = (
-                await self.embedding.embed_documents(
-                    self._embedding_client, [answer.question]
-                )
-            )[0]
-            if marginal_relevance:
-                matches, _ = self.texts_index.max_marginal_relevance_search(
-                    query_vector, k=_k, fetch_k=5 * _k
-                )
-            else:
-                matches, _ = self.texts_index.similarity_search(query_vector, k=_k)
+            matches = cast(
+                list[Text],
+                (
+                    await self.texts_index.max_marginal_relevance_search(
+                        self._embedding_client, answer.question, k=_k, fetch_k=5 * _k
```

Comment:
should we make this fetch_k=5 configurable in the function, with a default of 5?